src/Symfony/Component/HttpKernel/EventListener/LocaleListener.php

```diff
@@ -75,8 +79,11 @@ private function setRouterContext(Request $request)
     public static function getSubscribedEvents()
     {
         return array(
-            // must be registered after the Router to have access to the _locale
-            KernelEvents::REQUEST => array(array('onKernelRequest', 16)),
+            KernelEvents::REQUEST => array(
+                array('onKernelRequestEarly', 100),
```

Comment:
should we discuss the "100"? (I didn't check - just asking :) )

Reply:
Was matching the priority from #29186; only thing higher in core is `SessionListener` (and `DebugHandlersListener`). IMO these should be set as early as possible (so maybe even higher?).